### DISCUSSION TITLE
Ragnarok Online

### DISCUSSION THREAD

**Discussion by wilcoxlol** [GameServerManagers/LinuxGSM#3753] [Ideas]
Knowing that Ragnarok Online is an old game, it would be great if this type of server could be included :)

Excuse my English but my mother tongue is Spanish.

**Comment by h3o66** [comment]
There seems to be the server code here https://github.com/rathena/rathena , but imho it is hard to support a server that you need to compile yourself or the lgsm team needs to maintain
